lc-hax/Scripts/Listeners/EnemyListener.cs

```diff
@@ -0,0 +1,20 @@
+#pragma warning disable IDE1006
+
+using HarmonyLib;
+using System.Collections.Generic;
+
+[HarmonyPatch(typeof(EnemyAI))]
+class EnemyListener {
+    [HarmonyPatch(nameof(EnemyAI.OnDestroy))]
+    static void Prefix(ref EnemyAI __instance) {
+        if (ActiveEnemies.Contains(__instance)) _ = ActiveEnemies.Remove(__instance);
+    }
+
+    [HarmonyPatch(nameof(EnemyAI.Start))]
+    static void Postfix(ref EnemyAI __instance) {
+        if (!ActiveEnemies.Contains(__instance)) ActiveEnemies.Add(__instance);
```

Comment:
There's probably no need to check if the list contains two instances right?

Reply:
Better safe than sorry , sometimes it can be added twice.

Reply:
In that case, it might be better to use a HashSet which will do the similarity check for you.

Reply:
I also think just for consistency, we should move this to `Patches` instead. Especially, since it isn't 'listening' for anything in the traditional sense.

Reply:
Problem is that `Patches` are scripts that can be deleted without breaking the game. So maybe a new subfolder is needed.